src/components/Menu/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,207 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import Menu, { MenuProps } from '..'
+import { Box, Icon, Modal as SWUIModal, Touchable } from '../..'
+import Item from '../Item'
+
+export default {
+  component: Menu,
+  decorators: [
+    StoryComponent => (
+      <div style={{ marginBottom: '3em' }}>
```

Comment:
It is the margin applied to all stories. The idea is to show there's a dropdown then the user scrolls inside the story. I could customise the story heights but it seems a bit overkill?
Also the listing page height would be extremely long if I had more height? There is still the single story page if the user wants to see it properly?

What do you think @matthprost ?

Reply:
I believe as documentation reader I'd prefer having margin bottom right away to avoid scrolling down trough it, even tough it will increase the height of the story and makes the page longer. 

I think that making it without it makes the component looks like it is bottlenecked. That's a thing we use to do for other components such as `RichSelect` or `PhoneInput`.

Here is an example with `PhoneInput` of bad documentation experience if we do not use margin bottom:

https://user-images.githubusercontent.com/15812968/139822390-01b1f2d1-dc12-4c4b-8303-e37c5e2881d4.mov